src/pages/workspace/receiptPartners/WorkspaceReceiptPartnersPromotionBanner.tsx

```diff
@@ -0,0 +1,80 @@
+import React, {useCallback, useMemo} from 'react';
+import {View} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
+import Button from '@components/Button';
+import {Close} from '@components/Icon/Expensicons';
+import {PinkCar} from '@components/Icon/Illustrations';
+import useDismissedUberBanners from '@hooks/useDismissedUberBanners';
+import useLocalize from '@hooks/useLocalize';
+import usePermissions from '@hooks/usePermissions';
+import useResponsiveLayout from '@hooks/useResponsiveLayout';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {enablePolicyReceiptPartners} from '@libs/actions/Policy/Policy';
+import BillingBanner from '@pages/settings/Subscription/CardSection/BillingBanner/BillingBanner';
+import CONST from '@src/CONST';
+import type {Policy} from '@src/types/onyx';
+
+type WorkspaceReceiptPartnersBannerProps = {
+    policy: OnyxEntry<Policy>;
+    readOnly?: boolean;
+};
+
+function WorkspaceReceiptPartnersPromotionBanner({policy, readOnly}: WorkspaceReceiptPartnersBannerProps) {
+    const theme = useTheme();
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const StyleUtils = useStyleUtils();
+    const {shouldUseNarrowLayout} = useResponsiveLayout();
+    const {isBetaEnabled} = usePermissions();
+    const policyID = policy?.id;
+    const {setAsDismissed} = useDismissedUberBanners({policyID});
+    const {isDismissed} = useDismissedUberBanners({policyID: policy?.id});
```

Comment:
This is wrong, no? Shouldn't it be:
```suggestion
    const {setAsDismissed} = useDismissedUberBanners({policyID});
    const {isDismissed} = useDismissedUberBanners({policyID: policy?.id});
```
```suggestion
    const {isDismissed, setAsDismissed} = useDismissedUberBanners({policyID});
```